benchmarks/cpu_haswell.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+#SBATCH --nodes=1
+#SBATCH --constraint=haswell
+#SBATCH --time=15
+#SBATCH --qos=debug
+
+echo "Start time: $(date --iso-8601=seconds)"
+
+# Returns the number of seconds since unix epoch
+_time() {
+    echo $(date +%s.%N)
+}
+
+# Returns the elapsed time in seconds between start and end
+# setting scale=3 and dividing by 1 to limit to millisecond precision
+_elapsed_time () {
+    echo "scale=3; ($2 - $1) / 1" | bc
+}
+
+# Environement setup
+source /global/cfs/cdirs/desi/software/desi_environment.sh master
+
+export PATH=$(pwd)/bin:$PATH
+export PYTHONPATH=$(pwd)/py:$PYTHONPATH
```

Comment:
This expects the script to be launched from the gpu_specter top directory, which can clutter it up with slurm log files.  Consider adding an extra check about whether that has already been configured, e.g.
```
# see if gpu_specter is already in PYTHONPATH
python -c "import gpu_specter" 2&> /dev/null
if [ $? != 0 ]; then
    # try adding current directory
    export PATH=$(pwd)/bin:$PATH
    export PYTHONPATH=$(pwd)/py:$PYTHONPATH
    python -c "import gpu_specter" 2&> /dev/null

    # did that work?
    if [ $? != 0 ]; then
        echo "ERROR: gpu_specter not in PYTHONPATH; exiting
        exit 1
    fi
fi

```